alinka/widget/containers/main_body/content/settings/settings_tabs/support_center_tab/__init__.py

```diff
@@ -17,3 +20,14 @@ def __init__(self, parent: QWidget):
     @property
     def support_center_data(self):
         return self.support_center_data_group.support_center_data
+
+    @property
+    def is_valid(self) -> bool:
+        return bool(get_support_center_data())
+
+    @property
+    def error_message(self) -> str | None:
+        if not self.is_valid:
```

Comment:
Could be oneliner, but treat it as optional comment. :)

`return None if self.is_valid else (...)`